hardhat/hardhat.config.ts

```diff
@@ -127,15 +127,25 @@ task('create-user-policy', 'Creates a policy for the given user')
     const claimsAllowedUntil = args.claimsAllowedUntil
       ? parseISO(args.claimsAllowedUntil)
       : addDays(claimsAllowedFrom, 30);
-    const tx = await claimsManager.createPolicy(
+    const ipfsHash = args.ipfsHash || 'Qm' + randomBytes(22).toString('hex');
+
+    // We use a multicall in order to create the policy and announce its metadata in the same transaction
```

Comment:
Do you need to do that using mutlicall?

Reply:
Yeah that's how it will be called from the API3 market, because we want the `AnnouncedPolicyMetadata` event in the same block/transaction as the `CreatedPolicy` event.